core/store/orm/orm.go

```diff
@@ -1469,7 +1469,7 @@ type DialectName string
 
 const (
 	// DialectPostgres represents the postgres dialect.
-	DialectPostgres DialectName = "postgres"
+	DialectPostgres DialectName = "pgx"
```

Comment:
Does it automatically change DialectTransactionWrappedPosted as well? We want to make sure that tests are using the same driver as prod

Reply:
yes because we register that type with txdb now https://github.com/smartcontractkit/chainlink/blob/edebef90f0aa5529ac97ac82c28dd208ccf14010/core/internal/cltest/cltest.go#L143